homeassistant/helpers/entity.py

```diff
@@ -46,6 +46,7 @@
 from homeassistant.core import (
     CALLBACK_TYPE,
     Context,
+    Event,
     HassJobType,
     HomeAssistant,
     callback,
```

Comment:
Why is EventType import not removed?

Reply:
Probably missed one or two instances. I prepared the changes here a while ago. Will do a final pass after the current PRs are merged to catch the remaining ones if that's fine.